static/panes/selectiondag-view.js

```diff
@@ -0,0 +1,567 @@
+// Copyright (c) 2020, Dan Shechter
```

Comment:
Great stuff! Obviously quite a C&P but that's 100% ok, even encouraged. We can come back and try and find commonality later :) Thanks!